projects/plugins/debug-helper/modules/class-protect-helper.php

```diff
@@ -35,8 +35,9 @@ public function __construct() {
 			// Display a notice when this module overwrites Protect Status.
 			add_action( 'admin_notices', array( $this, 'display_protect_overwritten_notice' ) );
 
-			$option_name      = Automattic\Jetpack\Protect\Status::OPTION_NAME;
-			$option_time_name = Automattic\Jetpack\Protect\Status::OPTION_TIMESTAMP_NAME;
+			define( 'JETPACK_PROTECT_DEV__DATA_SOURCE', 'protect_report' );
```

Comment:
I think we should remove this from here so we can test the PR without having to change the code from it.

Reply:
I've updated the helper class to include [mock data for the Scan API](https://github.com/Automattic/jetpack/pull/26119/commits/de3e8c4eb507c9f2fec307660e4b858e9f73c673) to better test the plugin with either data source.